cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,79 @@
 /// <reference types='cypress' />
 
+const baseURL = 'https://demoqa.com/automation-practice-form';
```

Comment:
this is not a baseurl
https://docs.cypress.io/guides/references/configuration#Intelligent-Code-Completion